cloudinit/config/cc_ntp.py

```diff
@@ -80,6 +80,16 @@
             'confpath': '/etc/chrony/chrony.conf',
         },
     },
+    'centos': {
```

Comment:
Centos is a distro variant, so you can drop this section and only add a 'rhel' section.

```

Reply:
hm... i guess confusing as it is explicitly listed in 'distros' on line 27.

thats fine. i'll drop the centos.

Reply:
I agree.  I had to test it out on a centos container to be certain.